include/zephyr/devicetree.h

```diff
@@ -2203,6 +2203,14 @@
  */
 #define DT_REG_ADDR(node_id) DT_REG_ADDR_BY_IDX(node_id, 0)
 
+/**
+ * @brief 64-bit version of a DT_REG_ADDR macro.
+ *        Use it in cases when 64-bit address value may appear in Device Tree
+ * @param node_id node identifier
+ * @return node's register block address
```

Comment:
Please specify more clearly what the difference is between this value and the usual DT_REG_ADDR one.